Hestia/app/src/main/java/hestia/backend/ActivatorState.java

```diff
@@ -5,7 +5,7 @@
  * inferred using a custom JSON deserializer.
  * @see ActivatorDeserializer
  * @param <T> Type of the state of the activator. This can be a boolean (for a switch) or a float
- *           (for a slider)
+ *           (for a slider), or something else, depending on the Activator.
```

Comment:
Just wanted to show the flexibility of our program, the fact that we do not limit ourselves to only booleans or floats. I see that it sounds a bit odd. Perhaps is best if we remove the final part altogether.